Update tidb-lightning-faq.md from OnCALL (#6840)
pingcap/docs: tidb-lightning/tidb-lightning-faq.md

## `checksum failed: checksum mismatched remote vs local`

**Cause**: The checksum of a table in the local data source and the remote imported database differ. This error has several deeper reasons. You can further locate the reason by checking the log that contains `checksum mismatched`.

> The lines that contain `checksum mismatched` provides the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the import cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.

Reviewer: ```suggestion
The lines that contain `checksum mismatched` provides the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the imported cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.
```

maybe :joy: or "target cluster"

> The lines that contain `checksum mismatched` provides the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the target cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.

Reviewer 2: ```suggestion
The lines that contain `checksum mismatched` provide the information `total_kvs: x vs y`, where `x` indicates the number of key-value pairs (KV pairs) calculated by the target cluster after the import is completed, and `y` indicates the number of key-value pairs generated by the local data source.
```

> - If `x` is greater, it means that there are more KV pairs in the import cluster.

Reviewer 3: ```suggestion
- If `x` is greater, it means that there are more KV pairs in the target cluster.
```
should be `target` or `imported` here🤔

>     - It is possible that this table is populated before the import and therefore affects the data checksum. It is also possible that TiDB Lightning has previously failed and shut down, but did not restart correctly.

Reviewer 3: ```suggestion
    - It is possible that this table is not empty before the import and therefore affects the data checksum. It is also possible that TiDB Lightning has previously failed and shut down, but did not restart correctly.
```

- If `y` is greater, it means that there are more KV pairs in the local data source.
    - If the checksum of the target database is all 0, it means that no import has occurred. It is possible that the cluster is too busy to receive any data.

>     - It is possible that the exported data contains duplicate data, such as UNIQUE and PRIMARY KEYs with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.

Reviewer 2: ```suggestion
    - It is possible that the exported data contain duplicate data, such as the UNIQUE and PRIMARY KEYs with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.
```

>     - It is possible that the exported data contains duplicate data, such as unique and primary keys (UNIQUE and PRIMARY KEYs) with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.

Reviewer: ```suggestion
    - It is possible that the exported data contains duplicate data, such as UNIQUE and PRIMARY KEYs with duplicate values, or that the downstream table structure is case-insensitive while the data is case-sensitive.
```

in Chinese I want to make sure the terminology, in English I think it's no need

- Other possible reasons

>     - If the data source is machine-generated and not backed up by Dumpling, make sure the data conforms to the table limits, e.g., the AUTO_INCREMENT column needs to be positive and not 0.

Reviewer 2: ```suggestion
    - If the data source is machine-generated and not backed up by Dumpling, make sure the data conform to the table limits. For example, the AUTO_INCREMENT column needs to be positive and not 0.
```
